CriteoPublisherSdk/Tests/Utility/NetworkManager/CR_NetworkManagerSimulator.m

```diff
@@ -54,13 +54,14 @@ + (NSTimeInterval)interstitialTtl {
 }
 
 - (instancetype)initWithConfig:(CR_Config *)config {
-  MockWKWebView *webView = [[MockWKWebView alloc] init];
-  CR_ThreadManager *threadManager = [[CR_ThreadManager alloc] init];
-  CR_DeviceInfo *deviceInfo = [[CR_DeviceInfo alloc] initWithThreadManager:threadManager
-                                                               testWebView:webView];
-  NSURLSession *session = [NSURLSession
-      sessionWithConfiguration:[NSURLSessionConfiguration defaultSessionConfiguration]];
-  if (self = [super initWithDeviceInfo:deviceInfo session:session threadManager:threadManager]) {
+  // FIXME This is a different implementation and network manager should be a protocol. This would
```

Comment:
When is the FIXME planned to be worked on? JIRA?

Reply:
Added JIRA ticket